packages/prisma-faker/src/seed.ts

```diff
@@ -891,7 +891,8 @@ export function seed<
                 return [pool, tasks, { ...acc, [field.name]: id }]
               }
               case 'Int': {
-                throw new Error('Int @ids are not yet supported!')
+                // throw new Error('Int @ids are not yet supported!')
+                return [pool, tasks, { ...acc, [field.name]: id }]
```

Comment:
That won't work. I wanted to hold back, but the idea is that we set the `id` to task order, which is undoubtedly unique in the set.

Reply:
Again, temporary workaround because BYOI is not available yet. I rely on cleaning up the `sqlite` DB manually after the tests for now to generate Int IDs in order again (starting from 1), making it feel deterministic.